actors-akka/src/test/scala/reactivemongo/actors/testkit/package.scala

```diff
@@ -0,0 +1,9 @@
+package reactivemongo.actors
+
+import akka.actor.Actor
+import akka.testkit.{ TestActorRef => PekkoTestActorRef }
```

Comment:
!!

Reply:
Addressed in [Remove unnecessary import alias](https://github.com/ReactiveMongo/ReactiveMongo/pull/1245/commits/6ce14986627867731da13377e921d45995b3acf2)